mozilla/foundation.mozilla.org
# update the route for robots.txt

Closes https://github.com/mozilla/foundation.mozilla.org/issues/7736

This makes it so that running on anything except prod yields a locked down `robots.txt` response.

Testing: 
- check out this branch
- make sure your `.env` file has `ASSET_DOMAIN=` (without a value)
- `docker-compose up`
- visit http://localhost:8000/robots.txt

This should yield:
```
User-Agent: *
Disallow: /
```

- `docker-compose down`
- make sure your `.env` file has `ASSET_DOMAIN=foundation.mozilla.org`
- `docker-compose up`
- visit http://localhost:8000/robots.txt

This should yield:
```
User-Agent: *
Disallow: /*/artifacts/thimble
Disallow: /artifacts/thimble
```
